experimental/LieAlgebras/src/WeylGroup.jl

```diff
@@ -51,10 +51,28 @@ function isomorphism(::Type{FPGroup}, W::WeylGroup; set_properties::Bool=true)
   return MapFromFunc(W, G, iso, isoinv)
 end
 
+@doc raw"""
+    permutation_group(W::WeylGroup; set_properties::Bool=true) -> PermGroup
+
+Construct a group of type `PermGroup` that is isomorphic to `W`.
+
+Also see: [`isomorphism(::Type{PermGroup}, ::WeylGroup)`](@ref).
+"""
 function permutation_group(W::WeylGroup; set_properties::Bool=true)
   return codomain(isomorphism(PermGroup, W; set_properties))
 end
 
+# The degree of the chosen permutation group is optimal except in the cases E_6, E_7.
+# See Saunders, "Minimal faithful permutation degrees for irreducible Coxeter groups and binary polyhedral groups", 2014.
+@doc raw"""
+    isomorphism(::Type{PermGroup}, W::WeylGroup; set_properties::Bool=true) -> Map{WeylGroup, PermGroup}
+
+Construct an isomorphism between `W` and a group of type `PermGroup`.
+
+If set_properties = true, then the properties of the `PermGroup` which are already known for `W` are set.
+
+Also see: [`permutation_group(::WeylGroup)`](@ref).
+"""
```

Comment:
```suggestion
@doc raw"""
    isomorphism(::Type{PermGroup}, W::WeylGroup) -> Map{WeylGroup, PermGroup}

Construct an isomorphism between `W` and a group of type `PermGroup`.

The properties of the codomain group and the isomorphism are described in [`permutation_group(::WeylGroup)`](@ref).
"""
```
see above. Furthermore make this docstring analogous to the FPGroup case above

Reply:
I had copied the docstring from FPGroup and then removed the last sentence because `permutation_group(::WeylGroup)` does not actually describe any properties of the codomain group (except that it is isomorphic to `W`). However, I given your suggestion below, this makes sense.